TeachingRecordSystem/tests/TeachingRecordSystem.SupportUi.EndToEndTests/JourneyTests/EditRouteToProfessionalStatusTests.cs

```diff
@@ -885,23 +885,18 @@ public async Task EditStatus_Awarded_Continue_Exemption_Continue()
         await page.AssertOnRouteDetailPageAsync(qualificationId);
     }
 
-    [Fact]
+    [Fact(Skip = "Waiting for provider table to be populated")]
```

Comment:
Now that the implicit exemption is on the correct route, that route needs a provider to be specified, so skipping the test for now